RadixWallet/Features/SettingsFeature/Preferences/Preferences.swift

```diff
@@ -104,6 +105,13 @@ public struct Preferences: Sendable, FeatureReducer {
 				try await appPreferencesClient.updatePreferences(preferences)
 			}
 
+		case let .advancedLockToogled(isEnabled):
+			state.appPreferences?.security.isAdvancedLockEnabled = isEnabled
+			guard let preferences = state.appPreferences else { return .none }
+			return .run { _ in
+				try await appPreferencesClient.updatePreferences(preferences)
+			}
```

Comment:
Should probably handle the error. True that this can actually fail when the Profile failed to be saved, which would be a bigger issue, but still good practice to catch the error.